### PR TITLE
Add basic support for google game dashboard

### PR DESCRIPTION
**Pull request by QuackingCanary**
Add basic support for Google Game Dashboard that allows the user to switch between performance and battery saving modes

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
         android:fullBackupContent="@xml/backup_descriptor"
         android:icon="@drawable/logo_skyline"
         android:isGame="true"
+        android:appCategory="game"
```
Comment (lynxnb) [inline app/src/main/AndroidManifest.xml:21]: This needs to be tested, it was causing weird crashes on Adreno devices.

File: app/src/main/res/xml-v33/game_mode_config.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (lynxnb) [inline app/src/main/res/xml-v33/game_mode_config.xml:1]: Add an empty new line at the end of the file.

**Review by lynxnb — CHANGES_REQUESTED** [review]

**Review by lynxnb — APPROVED** [review]
Testing looks fine, the crash doesn't seem to happen anymore.
LGTM, thanks!

**Comment by Ishan09811** [comment]
> Testing looks fine, the crash doesn't seem to happen anymore. LGTM, thanks!

@lynxnb 
It means you are thinking about release in play store ? 😶

**Pull request merged**